packaging/portainer.spec

```diff
@@ -0,0 +1,115 @@
+#
+# spec file for package portainer
+#
+# Copyright (c) 2017 SUSE LINUX Products GmbH, Nuernberg, Germany.
```

Comment:
Not sure about the copyright and license in these file? Do we need to keep this?

Reply:
regarding copyright no problem at all, it is default for spec files so it can be removed

Reply:
Let's remove it then.